supportedBackends/tes/src/main/scala/cromwell/backend/impl/tes/TesAsyncBackendJobExecutionActor.scala

```diff
@@ -322,7 +320,6 @@ object TesAsyncBackendJobExecutionActor {
                      handle: StandardAsyncPendingExecutionHandle,
                      getTaskLogsFn: StandardAsyncPendingExecutionHandle => Future[Option[TaskLog]],
                      tellMetadataFn: Map[String, Any] => Unit,
-                     tellBardFn: StandardAsyncRunState => Unit,
```

Comment:
I'm going to create a ticket in this epic to track the fact that we've disabled Bard reporting here.